aikau/src/main/resources/alfresco/services/UploadService.js

```diff
@@ -183,10 +229,12 @@ define(["dojo/_base/declare",
 
             // Create a new dialog... the content is variable, but the widgets are fixed...
             this.progressDialog = new AlfDialog({
+               id: "ALF_UPLOAD_PROGRESS_DIALOG",
```

Comment:
Yes, the code in this service pre-dates a lot of the work that was done in the DialogService and it creates it's own dialog which is then shown/revealed as necessary. I did briefly experiment with trying to use the DialogService but it felt that it was out of scope for the task at hand. It is something that could potentially be worked on in the future though.